apps/web/components/booking/CancelBooking.tsx

```diff
@@ -31,7 +32,7 @@ type Props = {
       email: string;
       timeZone?: string;
     };
-    eventType: unknown;
+    eventType: any;
```

Comment:
This is not a good practice. You can get the eventType from `RouterOutput`. Please refer to another component where the eventType type is defined

Reply:
ok , thanks for reviewing ,will make the changes and reopen it.

Reply:
you can'r reopen it, so tag me here after finishing the work